### DISCUSSION TITLE
Wanted to buy the license but it says - "This is old license only for Pyarmor < 8.0. **It's could not be used in the latest Pyarmor**

### DISCUSSION THREAD

**Discussion by rishab4privacy** [dashingsoft/pyarmor#1035] [General]
Just came across this tool for obfuscation and it was an instant hit amongst our prod team and we played around with the free version for a bit. We then decided to go ahead and buy it but came across the mentioned disclaimer, when would the license for the latest version be out? or can I install a previous version of pyarmor and buy the license of pyarmor older than version 8?
![SCR-20230328-ds7](https://user-images.githubusercontent.com/107401922/228128076-5f98b749-3abc-4839-83aa-278d9942a7c0.png)

**Comment by jondy** [comment]
I suggest you purchase this old license and then using pyarmor 7.7.4 if you need use pyarmor immediately

Pyarmor 8.0 has a big change, it's totally different from Pyarmor 7.x, now only x86_64 arch is supported by Pyarmor 8.0

After pyarmor 8.0 is stable, old license could be upgraded to Pyarmor 8.0 Basic freely

For more information, refer to
https://pyarmor.readthedocs.io/en/stable/licenses.html
